app/views/appointments/search.html.erb

```diff
@@ -2,30 +2,40 @@
 
 <h1>Appointment search</h1>
 
-<div class="row form-group">
-  <div class="col-md-12">
-    <%= form_for @search, layout: :basic, method: 'GET', url: { action: :search } do |f| %>
-      <%= f.text_field :q, label: 'Search', class: 't-q', placeholder: 'Search by first name, last name, or booking reference.' %>
-      <%=
-        f.text_field(
-          :date_range,
-          class: 't-date-range',
-          data: {
-            module: 'date-range-picker',
-            config: {
-              autoUpdateInput: false,
-              locale: {
-                cancelLabel: 'Clear',
-                format: 'DD/MM/YYYY'
+<%= panel do %>
+  <div class="panel-body">
+    <%= form_for @search, method: 'GET', url: { action: :search }, html: { class: 'form-inline' } do |f| %>
+      <div class="form-group">
+        <%= f.label :q, 'Search term' %>
+        <%= f.text_field :q, use_label: false, class: 't-q appointment-search-term form-control', placeholder: 'Ref, Name or Guider' %>
```

Comment:
Can we drop the caps & abbreviation on these: `Reference, name or guider`  should fit?